api/v1alpha1/rsyncnodedatamovement_types.go

```diff
@@ -51,6 +51,14 @@ type RsyncNodeDataMovementSpec struct {
 	// DryRun specifies that this data movement request should show what would have been transferred without actually
 	// performing any copy operation.
 	DryRun bool `json:"dryRun,omitempty"`
+
+	// Cancel is a flag to indicate that this data movement should be
+	// terminated. This value should not be set on creation.
+	Cancel bool `json:"cancel,omitempty"`
+
+	// Use an alternative command to simulate data movement (e.g. sleep 120).
```

Comment:
Ohh I misunderstood - I thought you meant the --simulated option in the nnf-dm server. I'm okay if you want to remove this - I had implemented something similar with the data-movement-config. We could consolidate on one (or none).

Reply:
I think I might leave this in b/c I think it would be helpful for whoever is doing WLM integration to have a way to fake the DM so they can get the correct responses from the API. It was very helpful in my case. Thoughts?

Reply:
I'm good with that.